docs/static/Devel/Developers.md

```diff
@@ -61,7 +61,7 @@ consists in
 | `npm run docs-home`      | Generate the [JSDOC documentation landing page](../../jsdocConfig/jsdoc.home.json)                                                                                                                                                                                   |
 | `npm run docs`           | Run `npm run docs-shared`, `npm run docs-browser`, `npm run docs-node`, and `npm run docs-home`                                                                                                                                                                       |
 | `npm run dev-docs`       | Launch a watcher for generating and debugging the documentation. See [here](../../Readme.md) for more information                                                                                                                                                    |
-| `npm run host`           | host the bundle with an [ExpressAppWrapper](../../../bin/host.js) + a default game socket service. <br>http://locahost:8000/                                                                                                                                                                                    |
+| `npm run examples-back-end`           | run an http server (with some [string-replace](https://www.npmjs.com/package/string-replace-middleware))  + a game socket service. <br>http://locahost:8000/                                                                                                                                                                                    |
 | `npm run start`          | Run `npm run build-browser` and `npm run host`                                                                                                                                                                                                                        |
```

Comment:
This no longer true.

Reply:
What is no longer true ? actually this is what is doing the back end of the examples